### PR TITLE
Handle `undefined` and `null` in style objects

### LINKED ISSUE #180: Uncaught TypeError: Cannot read property 'config' of undefined
I am upgrading from 0.2.7 to 0.3.0 and I ran into an error:

> Uncaught TypeError: Cannot read property 'config' of undefined

This happens when I am using a `TransitionMotion` with a style object that has a key who's value is `undefined`, e.g.:

```js
{
  opacity: spring(0),
  extraText: undefined,
}
```

This is happening on https://github.com/chenglou/react-motion/blob/d478fe6ba1a5ba74ab364747f063e16fa8c3185f/src/updateTree.js#L38

I can side-step the issue by setting that value to something other than `undefined` or `null`, but it would be nice if I didn't have to. I believe this could be avoided by adding a couple of checks here and there, for example:

```js
if (!style[key].config) {
```

could be changed to

```js
if (!style[key] || !style[key].config) {
```

### PR DESCRIPTION
**Pull request by lencioni**
I am upgrading from 0.2.7 to 0.3.0 and I ran into an error:

> Uncaught TypeError: Cannot read property 'config' of undefined

This happens when I am using a `TransitionMotion` with a style object
that has a key who's value is `undefined`, e.g.:

  {
    opacity: spring(0),
    extraText: undefined,
  }

To make react-motion easier to work with, I have added guards in a
number of places.

Fixes #180.

### CODE REVIEW

**Comment by chenglou** [comment]
These checks should be `x == null` to avoid catching 0

**Comment by lencioni** [comment]
That's a fair point. I'll update my PR shortly.

File: test/stripStyle-test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import stripStyle from '../src/stripStyle';
```
Comment (chenglou) [inline test/stripStyle-test.js]: Neat, though can you pull this out into another PR once I merge this one?

Comment (lencioni) [inline test/stripStyle-test.js]: Should I just include the test for ignoring `undefined` in this PR and put the other 2 tests in a new PR?

Comment (chenglou) [inline test/stripStyle-test.js]: That works too, yeah.

**Comment by lencioni** [comment]
New version is ready to go. After you merge I'll submit a new PR for the additional tests.

File: test/updateTree-test.js [hunk 2]
```diff
@@ -65,6 +81,14 @@ describe('updateCurrentVelocity', () => {
       .toEqual({a: 0, b: 20.4});
   });
 
+  it('should handle `undefined`', () => {
+    const currentStyle = {a: undefined};
+    const currentVelocity = {a: undefined};
+    const style = {a: undefined};
+    expect(updateCurrentVelocity(FRAME_RATE, currentStyle, currentVelocity, style))
+      .toEqual({a: 0});
```
Comment (chenglou) [inline test/updateTree-test.js]: Humm, on second thought this should probably be `undefined`. My fault. Will reconsider this.

**Comment by chenglou** [comment]
Cool, done.

File: test/updateTree-test.js (hunk 2 above)
Comment (lencioni) [inline test/updateTree-test.js]: Yeah, I found this to be a little unexpected. :)

**Comment by lencioni** [comment]
My pleasure.

**Pull request merged**